cristin-person/src/main/java/no/unit/nva/cristin/person/update/CristinPersonPatchJsonCreator.java

```diff
@@ -184,6 +191,50 @@ private void addNviIfPresent() {
         }
     }
 
+    private void addContactDetailsIfPresent() {
+        if (input.has(CONTACT_DETAILS) && !input.get(CONTACT_DETAILS).isNull()) {
+            addTelephoneIfPresent();
+            addEmailIfPresent();
+            addWebPageIfPresent();
+        }
+    }
+
+    private void addTelephoneIfPresent() {
+        var contactDetails = input.get(CONTACT_DETAILS);
+
+        if (contactDetails.has(TELEPHONE)) {
+            if (contactDetails.get(TELEPHONE).isNull()) {
+                output.putNull(CRISTIN_TELEPHONE);
+            } else {
+                output.put(CRISTIN_TELEPHONE, contactDetails.get(TELEPHONE).asText());
+            }
+        }
```

Comment:
This is very verbose code for assigning one value to one field...
probably nothing you can change now.